docs/UserGuide.adoc

```diff
@@ -149,27 +160,37 @@ Returns `john` and `John Doe`
 * `find Betsy Tim John` +
 Returns any person having names `Betsy`, `Tim`, or `John`
 
-=== Deleting a person or a tag: `delete`
-==== Deleting a Person
+=== Deleting multiple persons or a tag: `delete`
+==== Deleting multiple persons
+
+Deletes the multiple specified persons from the address book. +
+Format: `delete [INDEX1], [INDEX2], [INDEX3]...` +
+Alternatives: `d` , `remove`
 
-Deletes the specified person from the address book. +
-Format: `delete INDEX` +
+Deletes the contacts which you haven't used the 'edit', 'photo' or 'find' command for K months. +
+Format: `delete old/K` +
```

Comment:
Perhaps could rename "K" to a more readable word

Reply:
oh yes